python/tk_framework_desktopserver/certificates.py

```diff
@@ -20,6 +20,7 @@
 from OpenSSL import crypto
 
 import sgtk
+import sgtk.util
```

Comment:
No need for this second import, the first one [does all the work](https://github.com/shotgunsoftware/tk-core/blob/master/python/tank/__init__.py#L117) 😄